.github/workflows/scrape-fj-manual.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           - name: sparse checkout country file directory
             uses: snow-actions/sparse-checkout@v1.1.0
             with:
+              token: ${{ secrets.WORKFLOW_TOKEN }}
               patterns: ./mcd-sel-fj
```

Comment:
Encountered authentication error (wrong password) on last run.
Amended to use Personal Access Token.